### PR TITLE
Asc 1647/master/use release version

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

**Comment by zreichert** [comment]
I will cut a dev release from this branch and we can watch it work.  Once we are satisfied we should release the real version of ZigZag and then use it here.  One that is complete this PR can merge and we will cut a real release.

File: pytest_zigzag/__init__.py [hunk 1]
```diff
@@ -152,14 +148,15 @@ def pytest_sessionfinish(session):
...
+        pytest_zigzag_config = _get_option_of_highest_precedence(session.config, 'pytest-zigzag-config')
+        if zz_option and pytest_zigzag_config:
             try:
                 junit_file_path = getattr(session.config, '_xml', None).logfile
+
                 # noinspection PyTypeChecker
                 # validate token
                 token = _validate_qtest_token(os.environ['QTEST_API_TOKEN'])
-                zz = ZigZag(junit_file_path, token, qtest_project_id, None)
+                zz = ZigZag(junit_file_path, pytest_zigzag_config, token)
```
Comment (ryan-rs) [inline pytest_zigzag/__init__.py]: This assumes that the `pytest_zigzag_config` file contains both "pytest-zigzag" and "zigzag" configuration settings which is not guaranteed. The documentation should be updated to reflect the best practice for invoking "zigzag" with the right configuration settings.

Comment (ryan-rs) [inline pytest_zigzag/__init__.py]: This assumes that the `pytest_zigzag_config` file contains both "pytest-zigzag" and "zigzag" configuration settings which is not guaranteed. The documentation should be updated to reflect the best practice for invoking "zigzag" with the right configuration settings.

Comment (zreichert) [reply pytest_zigzag/__init__.py]: Filed this ticket https://rpc-openstack.atlassian.net/browse/ASC-1674

File: setup.cfg [hunk 2]
```diff
@@ -3,16 +3,16 @@ current_version = 1.0.0
 commit = True
 tag = False
 parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-(?P<release>[a-z]+)(?P<build>\d+))?
-serialize =
-        {major}.{minor}.{patch}-{release}{build}
-        {major}.{minor}.{patch}
+serialize =
```
Comment (ryan-rs) [inline setup.cfg]: Added an extra space after `=` sign.

File: setup.cfg [hunk 3]
```diff
@@ -3,16 +3,16 @@ current_version = 1.0.0
...
+	{major}.{minor}.{patch}-{release}{build}
+	{major}.{minor}.{patch}
 
 [bumpversion:part:release]
 optional_value = prod
 first_value = dev
-values =
-        dev
-        prod
+values =
```
Comment (ryan-rs) [inline setup.cfg]: Added an extra space after `=` sign.

**Pull request merged**